28-04/task_2/cleanfile.rb

```diff
@@ -0,0 +1,14 @@
+class Cleanfile
+  attr_accessor :file_name
+
+  def initialize
+    @file_name = "palindrom.txt"
+  end
+
+  def open
+    File.open(@file_name).each do |line|
```

Comment:
you can `file_name` method here instead of `@file_name`. Usually it's a better approach to call instance variablew through a getter method